Kryptonite/Silo.swift

```diff
@@ -310,7 +310,9 @@ class Silo {
             
             var pgpPublicKey:Data?
             if let pgpUserID = meRequest.pgpUserId {
-                pgpPublicKey = try keyManager.loadPGPPublicKey(for: pgpUserID).packetData
+                let message = try keyManager.loadPGPPublicKey(for: pgpUserID)
+                pgpPublicKey = message.packetData
+                log("pgp public key:\n\(message.toString())", .warning)
```

Comment:
change to `.info` log level?

Reply:
just removed the log line, it was for testing